cmd/initdb/main.go

```diff
@@ -25,27 +26,58 @@ var entries = []Entry{
 }
 
 const fmtStr = "insert into users (first_name, last_name, email) values ('%s', '%s', '%s')"
+const tableQuery = `SELECT tablename FROM pg_catalog.pg_tables where tablename = $1`
+
+func checkTableExists(db *sql.DB) (bool, error) {
+	rows, err := db.Query(tableQuery, "users")
+	if err != nil {
+		return false, err
+	}
+	defer rows.Close()
+	for rows.Next() {
```

Comment:
Do you have to walk through the rows or can you just close?

Reply:
it could return no rows and it wouldn't be an error, so I think so?